packages/replay-next/src/contexts/FocusContext.tsx

```diff
@@ -129,7 +135,7 @@ export function FocusContextRoot({ children }: PropsWithChildren<{}>) {
   }, [loadedRegions]);
 
   const updateFocusRange = useCallback(
-    async (range: TimeStampedPointRange | null, options: UpdateOptions) => {
+    async (range: TimeStampedPointRange, options: UpdateOptions) => {
```

Comment:
nit: Changing the `range` param (and the `range` state) to not be nullable means that the conditional not-null checks on line 144 and line 146can be removed.